src/Components/HomeScreenComponents/PlayerDetails.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+
+interface myProps{
+    playerNumber: number
+}
+
+function PlayerDetails(props: myProps) {
+    return (
+        <div>
+            <label htmlFor="playerName">Player {props.playerNumber}: </label>
+            <input type="text" id="playerName" placeholder="Name"/>
```

Comment:
This component will be rendered more than once on the home page so its `id` prop should be unique (see [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id)).  This can be achieved using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals):
```js
id = `player-${props.playerNumber}-name`
```